pwnlib/util/proc.py

```diff
@@ -99,8 +99,11 @@ def name(pid):
         Name of process as listed in ``/proc/<pid>/status``.
 
     Example:
-        >>> pid = pidof('init')[0]
-        >>> name(pid) == 'init'
+        >>> pid = 1
+        >>> n = name(pid)
+        >>> n
+        'systemd'
```

Comment:
This will break when running `make -C travis/docker` since Docker uses `tini` init when run with `--init` (as we do, so that this test would pass).

Reply:
So this could just have `doctest: +SKIP`?

Reply:
If it causes issues I'd prefer to just re-write the test to do something else (e.g. create a process, e.g. p=process('sh'); name(p.pid) == 'sh'; etc.